### DISCUSSION TITLE
Member deletion?

### DISCUSSION THREAD

**Discussion by filippog** [Netflix/dispatch#2652] [Q&A] [answered]
Hello,
I'm [deploying Dispatch and Wikimedia Foundation](https://phabricator.wikimedia.org/T313228) and integrating it with our SSO. Members are created on first login automatically as we're using HTTP header auth from #2494. We're also syncing other information in the background like real name from LDAP using the API. I was looking for a way to delete organization members (e.g. in our case users no longer active) though I couldn't find the method to do so. Is that something that's supported and/or can be added? Thank you!

**Comment by kevgliss** [comment, marked as answer]
Hi @filippog, I don't believe we currently support the deleting of members. We are in the same boat as our SSO creates new members if they don't already exist. 

I think we'd be open to supporting the deletion of members, but the one question I would have is how to handle things that are assigned to users that are removed. Many applications re-assign them to a "deleted" user or something similar or mark the user as 'inactive,' but I honestly haven't thought about it too deeply. 

By relying on SSO to prevent access, leaving the user object wasn't a huge deal for us.

**Reply by filippog** [reply]
Thank you for the quick answer and the insight @kevgliss !

I didn't think about the repercussions of deleting members and how to assign things -- that makes sense! Indeed using SSO to gate login and leaving behind inactive members isn't a huge deal.
